daemon/client/client.go

```diff
@@ -194,6 +194,33 @@ func main() {
 	fmt.Printf("---> %+v\n", details)
 	fmt.Println()
 
+	// BOARDS ATTACH
+	fmt.Println("=== calling BoardAttach(serial:///dev/ttyACM0)")
+	boardattachresp, err := client.BoardAttach(context.Background(), &rpc.BoardAttachReq{
+		Instance:   instance,
+		BoardURI:   "/dev/ttyACM0",
+		SketchPath: "/home/riccardo/Arduino/MyFirstSketch",
```

Comment:
this path is wrong, use the command line argument

Reply:
ok